devops/ansible/roles/glove/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Address the service will listen on
+glove_address: 0.0.0.0:8080
+
+# URL to a substrate node endpoint
+# See https://wiki.polkadot.network/docs/maintain-endpoints for more information.
+glove_node_endpoint: "wss://rococo-rpc.polkadot.io"
+
+# Secret phrase for the Glove proxy account.
+# See https://wiki.polkadot.network/docs/learn-account-advanced#derivation-paths for more details.
+glove_proxy_secret_phrase: "\"caution juice atom organ advance problem want pledge someone senior holiday very/0\""
```

Comment:
Can we perhaps rename this file to `.../test/main.yml` or `test.yml`, so that it's clearer this is a test key?

Reply:
No, we cannot. Not sure if it even would work. However, in Ansible, you don't expect defaults to be production quality values, so it's expected that this is tested values. In general, I plan to add DevOps dedicated documentation, and going to make clear.